.prettierrc

```diff
@@ -1,7 +1,7 @@
 {
-  "trailingComma": "es5",
-  "semi": false,
-  "singleQuote": false,
+  "trailingComma": "all",
+  "semi": true,
+  "singleQuote": true,
```

Comment:
can you please revert these changes, no semicolon and double quote is preferred.
and add the bracketSmameLine to false and completely remove biomejs from the repo.
thanks.

Reply:
1. Will not include removal of biomejs in this PR. Can you create issue for me and will do it on another PR? 
2. Personally prefer semicolon over no semicolon. It's give clear visual representation of end of command. Can we add semi? 
3. singleQuote -> ✅